src/test/java/com/yelp/nrtsearch/server/grpc/LuceneServerTest.java

```diff
@@ -277,6 +277,24 @@ public void testStartShard() throws IOException {
     assertTrue(!reply.getSegments().isEmpty());
   }
 
+  @Test
+  public void testStartIndex() {
+    String testIndex = grpcServer.getTestIndex();
+    LuceneServerGrpc.LuceneServerBlockingStub blockingStub = grpcServer.getBlockingStub();
+    // create the index
+    blockingStub.createIndex(CreateIndexRequest.newBuilder().setIndexName(testIndex).build());
+    try {
+      // start the index
+      String emptyTestIndex = "";
```

Comment:
Also test cases where the indexName is not set in addition to testing for empty strings.

Reply:
Added another test case: launch the index with empty string before creating a index.

Reply:
Can we add these two different test cases in two different functions? The create index test can be moved into a test called `testCreateIndexWithEmptyString`

Reply:
It sounds a good idea, I splitted the unit tests into two.
For the naming, I name it more specifically: `testCreateAndStartIndexWithEmptyString` since we are actually testing the start index functionality.

Reply:
Ohh my bad @swekannan I thought we were testing the createIndex as well. Yeah just test for startIndex is fine if we have not changed anything in the create.